static/src/javascripts/projects/commercial/modules/dfp/track-ad-render.ts

```diff
@@ -1,4 +0,0 @@
-import { waitForAdvert } from './wait-for-advert';
```

Comment:
`track-ad-render` was wrapping `wait-for-advert` with the timing stuff, it was only still left behind for `im` so can be removed now.